src/azul/project/hca/transformers.py

```diff
@@ -168,15 +172,20 @@ def create_documents(self,
                          manifest: List[JSON],
                          metadata_files: Mapping[str, JSON]
                          ) -> Sequence[ElasticSearchDocument]:
+        # Create a bundle object.
```

Comment:
I think this is clear enough from the code and doesn't warrant adding a comment.

Reply:
Got it. Will remove this in the next commit.